connect/runtime/src/test/java/org/apache/kafka/connect/runtime/ErrorHandlingTaskTest.java

```diff
@@ -482,6 +475,13 @@ private void assertSinkMetricValue(String name, double expected) {
         assertEquals(expected, measured, 0.001d);
     }
 
+    private void verifyInitializeTask() {
```

Comment:
This method appears to only apply to sink tasks; can we rename it to something like `verifyInitializeSink`?